code/_onclick/cyborg.dm

```diff
@@ -39,7 +39,7 @@
 		CtrlClickOn(A)
 		return
 
-	if(stat || lockcharge || weakened || stunned || paralysis)
+	if(incapacitated())
```

Comment:
Oh, that was in the other part of the refactor - I'll bring it over, since I forgot that